requirements/test.in

```diff
@@ -10,6 +10,7 @@ ddt
 edx-lint
 isort
 factory_boy>=2.6.1,<3.0.0
+rest-condition
```

Comment:
Add it in test requirements so that tests against it can still be run. If we can completely drop it then even better.